src/util-decode-mime.c

```diff
@@ -2547,6 +2540,13 @@ int MimeDecParseComplete(MimeDecParseState *state)
         return ret;
     }
 
+#ifdef HAVE_NSS
+    if (state->md5_ctx) {
+        unsigned int len = 0;
```

Comment:
This `len` feels weird... Is this something from NSS ?

Reply:
Yeah. I think it returns to you the size of the digest.